ably/docs
# [WEB-346] Add internal links on the website to pub-sub concept

Explain why this change is being made.
----
Link to pub/sub concept page was not present in the content of the website

Explain how this is accomplished.
----
Added pub/sub links to content of the textile files


How do you manually test this?
----
Navigate to `/documentation/core-features/pubsub`
Navigate to `/tutorials/publish-subscribe`
Navigate to `/concepts/mqtt`
Navigate to `/documentation/mqtt`
Navigate to `/concepts/websub`

Look for `publish-subscribe` or `pub/sub` should to navigate to `/concepts/pub-sub`

Merge/Deploy Checklist
----


Frontend Checklist
----

- [ ] Have you compared with the initial design / fit our [brand guidelines](https://brand.ably.com/)?
